librz/core/cmd/cmd_analysis.c

```diff
@@ -7998,3 +7941,43 @@ RZ_IPI RzCmdStatus rz_analyze_value_to_maps_handler(RzCore *core, int argc, cons
 	rz_core_analysis_value_pointers(core, state->mode);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_info_show_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_STANDARD:
+	case RZ_OUTPUT_MODE_JSON:
+		cmd_address_info(core, core->offset, state);
+		break;
+	default:
+		rz_warn_if_reached();
+		break;
+	}
```

Comment:
```suggestion
   cmd_address_info(core, core->offset, state);
```

probably this can be done this way

Reply:
Thanks for your review.